packages/material-renderers/src/complex/ConfirmDialog.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@mui/material';
+
+export interface ConfirmDialogProps {
+  open: boolean;
+  handleClose: () => void;
+  confirm: () => void;
+  cancel: () => void;
+  id: any;
+}
+
+export const ConfirmDialog = ({
```

Comment:
Could we name this in a way that immediately makes clear what kind of confirmation this dialog is for? E.g. `TabSwitchConfirmDialog`.